scripts/find_diverged_forks.py

```diff
@@ -2,6 +2,10 @@
 import argparse
 from github import Github
 
+import os
+
+os.system("echo \""+os.path.abspath(__file__).replace(os.getcwd(), "")+"\" >> ~/.paas-script-usage")
```

Comment:
Again, seems a bit weird to shell out to echo to append to a file.

Reply:
Same explanation as for Ruby, combined with me not knowing Python at all.

Reply:
```python
script_path = os.path.relpath(__file__)
with open(os.path.expanduser('~/.paas-script-usage'), 'a') as f:
  f.write(script_path + "\n")
```